helm/hpcc/templates/_helpers.tpl

```diff
@@ -193,36 +192,35 @@ Pass in root and includeLabels (optional)
   {{- end }}
  {{- end }}
 {{- end }}
+{{- /*Backwards compatibility: when no data planes defined in values, create default pvc*/ -}}
+{{- if and (or (has "data" $includeLabels) (has "" $includeLabels)) (not $dataStorage.plane) }}
```

Comment:
I don't think this is the right place for this (in this generalized addVolumeMounts function).
and afaict, it's going to cause calls with includeLabels containing "data" to generate a plane with "data" in main loop and this one as well.

It should only generate this default one if no dataStorage.plane has been defined, but it should do that in addDataVolumeMount 

The special empty ("") case is already being handled by doesStorageLabelsMatch and "" passed in addDataVolumeMount

Reply:
I've commented in the code why this is necessary.